source/delegate/reports/SPEC.md

```diff
@@ -0,0 +1,107 @@
+# Report: Delegate
```

Comment:
is this file necessary?

Reply:
removing its part of the delegate-report branch